### PR TITLE
Add Twitch to video embed sites, fixing #1750

### PR DESCRIPTION
**Pull request by euantorano**
Adds support for Twitch as a video embed site, fixing #1750.

### CODE REVIEW

**Comment by JN-Jones** [comment]
@Ben-Conway can you take a look at this?

File: jscripts/bbcodes_sceditor.js [hunk 1]
```diff
@@ -529,6 +534,7 @@ $(document).ready(function($) {
 						'<option value="veoh">' + editor._('Veoh') + '</option>' +
 						'<option value="vimeo">' + editor._('Vimeo') + '</option>' +
 						'<option value="youtube">' + editor._('Youtube') + '</option>' +
+						'<option value="twitch">' + editor._('Twitch') + '</option>' +
```
Comment (JN-Jones) [inline jscripts/bbcodes_sceditor.js]: Though it doesn't seem to have that much of an impact this variable should also be added to `global.lang.php` as `editor_twitch` like the rest  to make it translatable.

**Comment by JN-Jones** [comment]
Apart from the language string this is fine and can be merged.

**Comment by euantorano** [comment]
I'll add that tonight.

**Comment by euantorano** [comment]
@JN-Jones Fixed.

**Pull request merged**